docs/documentation/functions/aria-suite/aria-operations-networks/Test-AriaNetworksInternalAuthentication.md

```diff
@@ -0,0 +1,82 @@
+# Test-AriaNetworksInternalAuthentication
+
+## Synopsis
+
+Check authentication to the VMware Aria Operations for Networks instance for using the internal API.
+
+## Syntax
+
+```powershell
+Test-AriaNetworksInternalAuthentication [-server] <String> [-user] <String> [-pass] <String>
+ [<CommonParameters>]
+```
+
+## Description
+
+The Test-AriaNetworksInternalAuthentication cmdlet checks authentication to a VMware Aria Operations for Networks

```

Comment:
```suggestion
The `Test-AriaNetworksInternalAuthentication` cmdlet checks authentication to a VMware Aria Operations for Networks

```

Reply:
Fixed in https://github.com/vmware/power-validated-solutions-for-cloud-foundation/commit/42261f7bbfc80fe8925418f1c844be4895a4d95f